cmd/basecli/commands/cmds.go

```diff
@@ -10,48 +10,56 @@ import (
 
 	"github.com/tendermint/light-client/commands"
 	txcmd "github.com/tendermint/light-client/commands/txs"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+	cmn "github.com/tendermint/tmlibs/common"
 
 	btypes "github.com/tendermint/basecoin/types"
 )
 
-/*** Here is the sendtx command ***/
+/******** SendTx *********/
 
+// SendTxCmd is CLI command to send tokens between basecoin accounts
 var SendTxCmd = &cobra.Command{
 	Use:   "send",
 	Short: "send tokens from one account to another",
 	RunE:  doSendTx,
 }
 
+//nolint
 const (
-	ToFlag       = "to"
-	AmountFlag   = "amount"
-	FeeFlag      = "fee"
-	GasFlag      = "gas"
-	SequenceFlag = "sequence"
+	FlagTo       = "to"
```

Comment:
Sure, I guess this is the more standard naming?

We should make sure we put `Flag` first in all repos then.